src/main/java/RegisterAndLogin/Test2_failInOtherProject.java

```diff
@@ -0,0 +1,40 @@
+package RegisterAndLogin;
```

Comment:
This file has only one test and the test is empty, if the test is removed then this file should be removed too